package.json

```
@@ -54,7 +54,7 @@
     "build:ts": "tsc -b",
     "cache": "node scripts/ava-cache.js",
     "clean": "npm-run-all clean:*",
-    "clean:packages": "rimraf \"packages/!(connector-edge|extension-vscode)/{dist,package-lock.json,yarn.lock}\"",
+    "clean:packages": "rimraf \"packages/!extension-vscode/{dist,package-lock.json,yarn.lock}\"",
```

This should be able to go back to `*` as `extension-vscode` hasn't needed to be in this list since we removed `node_modules` from the targets.
```suggestion
    "clean:packages": "rimraf \"packages/*/{dist,package-lock.json,yarn.lock}\"",
```